include/swift/Remote/MetadataReader.h

```diff
@@ -264,6 +264,14 @@ class MetadataReader {
     return start;
   }
 
+  /// Given a pointer to an address, attemp to read the pointed value.
+  llvm::Optional<StoredPointer> readPointedValue(StoredPointer Address) {
+    StoredPointer PointedVal;
+    if (!Reader->readInteger(RemoteAddress(Address), &PointedVal))
+      return llvm::None;
+    return llvm::Optional<StoredPointer>(PointedVal);
```

Comment:
You should be able to just `return PointedVal;` here.  Also, you don't need the `llvm::` qualification.

Reply:
I'm going to do a full pass on this file next to remove the `llvm::` qualification everywhere. But I guess I can remove it here to begin.